x/stakeibc/keeper/community_pool.go

```diff
@@ -224,33 +224,57 @@ func (k Keeper) RedeemCommunityPoolTokens(ctx sdk.Context, hostZone types.HostZo
 	return nil
 }
 
-// Using tokens in the CommunityPoolReturnIcaAddress, trigger ICA tx to fund community pool
-// Note: The denom of the passed in token has to be the denom which exists on the hostZone not Stride
-func (k Keeper) FundCommunityPool(
+// Builds a msg to send funds to a community pool
+// If the community pool treasury address is specified on the host zone, the tokens are bank sent there
+// Otherwise, a MsgFundCommunityPool is used to send tokens to the default community pool address
+func (k Keeper) BuildFundCommunityPoolMsg(
 	ctx sdk.Context,
 	hostZone types.HostZone,
-	token sdk.Coin,
+	tokens sdk.Coins,
 	senderAccountType types.ICAAccountType,
-) error {
-	fundCoins := sdk.NewCoins(token)
-
-	// Get the depositor ICA address based on the account type
-	var depositor string
+) (fundMsg []proto.Message, err error) {
+	// Get the sender ICA address based on the account type
+	var sender string
 	switch senderAccountType {
 	case types.ICAAccountType_COMMUNITY_POOL_RETURN:
-		depositor = hostZone.CommunityPoolReturnIcaAddress
+		sender = hostZone.CommunityPoolReturnIcaAddress
 	case types.ICAAccountType_WITHDRAWAL:
-		depositor = hostZone.WithdrawalIcaAddress
+		sender = hostZone.WithdrawalIcaAddress
 	default:
-		return errorsmod.Wrapf(types.ErrICATxFailed,
+		return nil, errorsmod.Wrapf(types.ErrICATxFailed,
 			"fund community pool ICA can only be initiated from either the community pool return or withdrawal ICA account")
 	}
 
-	var msgs []proto.Message
-	msgs = append(msgs, &disttypes.MsgFundCommunityPool{
-		Amount:    fundCoins,
-		Depositor: depositor,
-	})
+	// If the community pool treasury address is specified, bank send there
+	if hostZone.CommunityPoolTreasuryAddress != "" {
+		fundMsg = []proto.Message{&banktypes.MsgSend{
```

Comment:
So I assume that we have verified that the Treasury module account they are using can accept normal bank sends then.  And the only addresses which can send are the withdrawal address or the community pool return ICA both of which actually exist on the DYDX chain so bank sends will work.  I think this looks good